src/main/webapp/jsp/instructorFeedbacks.jsp

```diff
@@ -5,6 +5,8 @@
 <%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
 <%@ taglib tagdir="/WEB-INF/tags/instructor/feedbacks" prefix="feedbacks" %>
 <c:set var="jsIncludes">
+  <script type="text/javascript" src="<%= FrontEndLibrary.MOMENT %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.MOMENT_TIMEZONE %>"></script>
```

Comment:
Is this needed?

Reply:
@whipermr5 Yes, without adding these, sessions page throws an error that "moment" reference not found.

Reply:
I'm referring to the timezone library, since we're not using `moment.tz` here.

Reply:
@whipermr5 Yes, you're right. It is not needed.

Reply:
Done.